language/stackless-bytecode/bytecode-to-boogie/test_mvir/test-special-instr.mvir

```diff
@@ -1,6 +0,0 @@
-module TestSpecialInstr {
-	public test_create_account() {
-        create_account(0x42);
```

Comment:
Keep with `LibraAccount.create_account`?

Reply:
does it make sense? I think this was testing the bytecode not the behavior of create_account. Wouldn't that be like testing a call?
I am not too familiar with this part of the codebase and if you know more I am definitely open to add that and understand what the test is doing

Reply:
I think this was testing that the `create_account` bytecode gets translated into the appropriate Boogie model. I'm not sure what that model should look like now that the native `LibraAccount.save_account`is the magic thing, but @emmazzz would surely know.

Reply:
I'll followup then

Reply:
@dariorussi : I suggest you revert your changes to the file bytecode_instrs.bpl so we keep around the model of CreateAccout. This model will now be used for modeling the CreateAccount native call.  We will handle that later.  You should go ahead with this PR.

Reply:
@shazqadeer I assume you are suggesting to go down @sblackshear's suggestion of checking the test to be a call to `LibraAccount.create_new_account()` right?
the bytecode is gone so I cannot leave the test there

Reply:
I am proposing you delete the test.  In fact, you do everything that you have done except revert your changes to bytecode_instrs.bpl.

If you simply change the test and make a call to the native method instead of the bytecode instruction, the test is unlikely to do the correct thing because the bytecode->boogie translator must also be updated to recognize that a call to a native method is being made, a Boogie model of the native method must be handwritten, and the call must be wired in the translator to a call to the model.  I am suggesting that we do this work as part of a separate PR because I don't yet fully understand the wiring of native methods in the translator.   I will look into this next week and do something about it.

Reply:
thanks @shazqadeer got it now. I never reverted the change that is why I missed the point :)
I basically leave things as I did by leaving the test out and we'll get to this when we get to this